sass/_toolkit.scss

```diff
@@ -464,6 +464,10 @@
 
 .nypl-results-item {
   @include results-item;
+
```

Comment:
Is there a way to put this in one of the partials? _toolkit convention is that this rule would go into the mixin...

Reply:
Do you mean a separate file partial? Or a mixin partial?

Reply:
yeah, could you put the class and rule in the mixin `results-item`?